Resources/Prototypes/_White/Entities/Clothing/Masks/masks.yml

```
@@ -0,0 +1,13 @@
+  parent: ClothingMaskBase
+  id: ClothingMaskToyVampireFangs
+  name: toy vampire fangs
+  description: Plastic toy vampire fangs made especially for the spooky day and maybe not only for this day. They smell of blood...
+  components:
+  - type: Sprite
+    sprite: _White/Clothing/Mask/toy_vampire_fangs.rsi
+  - type: Clothing
+    sprite: _White/Clothing/Mask/toy_vampire_fangs.rsi
+  - type: Tag
+    tags:
+    - IPCMaskWearable
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)